addons/a11y/src/a11yRunner.ts

```diff
@@ -41,13 +36,14 @@ const run = async (storyId: string) => {
       channel.emit(EVENTS.RUNNING);
       const axe = await import('axe-core');
 
-      const { element = getElement(), config, options = {} } = input;
+      const { element = 'root', config, options = {} } = input;
+      const htmlElement = document.querySelector(`#${element}`);
```

Comment:
Sorry, perhaps I wasn't clear enough in my comment. Here's what I was after:

```suggestion
      const { element = '#root', config, options = {} } = input;
      const htmlElement = document.querySelector(element);
```

This way, consumers can pass, for example, `element: '[data-testid="whatever"]'` instead of being limited to passing an `id`.

Reply:
Oh ok, yes I didn't understand.
I was looking to update the addon documentation, but it seems that this element option was meant to be a selector and not an id :D 

[Current documentation](https://storybook.js.org/addons/@storybook/addon-a11y)
<img width="541" alt="image" src="https://user-images.githubusercontent.com/2874505/165828989-5538f4a5-987c-46f5-b946-e4803c49586e.png">

Reply:
Suggestion commited, thanks again @kylegach